### PR TITLE
Doc: Fix link format for proper rendering

### PR DESCRIPTION
**Pull request by karenzone**

### CODE REVIEW

**Review by kilfoyle — DISMISSED** [review]
LGTM! 🎸

File: serverless/pages/ingest-your-data-ingest-data-through-integrations-logstash.asciidoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ such as data routed to multiple destinations or when you need to make your data
 {ls} is a powerful, versatile ETL (Extract, Transform, Load) engine that can play an important role in organizations of all sizes.
 Some capabilities and features for large, self-managed users aren't appropriate for {serverless-short}.
 
-You'll use the {ls} {logstash-ref}/plugins-outputs-elasticsearch.html[`((es)) output plugin`] to send data to {es3}.
+You'll use the {ls} {logstash-ref}/plugins-outputs-elasticsearch.html[`{es} output plugin`] to send data to {es3}.
```
Comment (karenzone) [inline serverless/pages/ingest-your-data-ingest-data-through-integrations-logstash.asciidoc:24]:
```suggestion
You'll use the {ls} {logstash-ref}/plugins-outputs-elasticsearch.html[{es} output plugin] to send data to {es3}.
```

**Review by leemthompo — APPROVED** [review]
🚗

**Comment by karenzone** [comment]
17484

**Pull request merged**